Resources/Prototypes/Catalog/Fills/Boxes/emergency.yml

```diff
@@ -186,6 +198,11 @@
     - id: Flare
     - id: FoodSnackNutribrick
     - id: DrinkWaterBottleFull
+  - type: Sprite
+    layers:
+    - state: box_hug
+    - state: heart
+    - state: nitrogentank
```

Comment:
Not sure about this one, it looks kinda weird. I think it's okay to just leave the boxhug one with the same look

![image](https://github.com/user-attachments/assets/1b8ffd5b-a86e-4815-b3c7-d6cf12aa53c6)